include/EVT/utils/log.hpp

```diff
@@ -0,0 +1,69 @@
+#ifndef EVT_LOG_
+#define EVT_LOG_
+
+#include <cstdint>
+#include <EVT/io/UART.hpp>
+#include <EVT/dev/RTC.hpp>
+
+namespace dev = EVT::core::DEV;
+
+namespace EVT::core::log {
+    class Logger {
```

Comment:
Minor stylistic thing, we don't have the classes indented inside of the name space, so we do
```
namespace EVT::core::log {

class Logger {
public:
    void log(...);
};

}
```

I would also check the output of cpplint to make sure you are matching the syle guide we follow.